src/repository/records.repository.js

```diff
@@ -0,0 +1,216 @@
+import { PrismaClient } from '@prisma/client';
+
+export class RecordsRepository {
+   constructor(prisma) { 
+        this.prisma = prisma;
+    }
+ /**
+ *운동 기록 생성
+ * @param {number} userId - 사용자 ID
+ * @param {number} groupId - 그룹 ID
+ * @param {string} nickname - 닉네임
+ * @param {string} exercise_type - 운동 종류
+ * @param {string} description - 설명
+ * @param {number} time - 운동 시간
+ * @param {number} distance - 운동 거리
+ * @param {string} password - 비밀번호
+ */
+    createRecord = async (
+        userId,
+        groupId,
+        nickname,
+        exercise_type,
+        description,
+        time,
+        distance,
+        password,
+        photos,
+    ) => {
+        const record = await this.prisma.record.create({
+            data: {
+                user_id: userId,
+                group_id: groupId,
+                nickname,
+                exercise_type,
+                description,
+                time,
+                distance,
+                password,
+                
+                images: {
+                    create: photos.map(photoPath => ({
+                        name: photoPath.substring(photoPath.lastIndexOf('/') + 1),
+                        path: photoPath,
+                    })),
+                },
+            },
+            include: {
+                images: true,
+            }
+        });
+        return record;
+    };
```

Comment:
```suggestion
    createRecord = async (recordData) => {
        const record = await this.prisma.record.create({
            data: recordData, // 전달받은 객체를 그대로 사용!
            include: {
                images: true,
            },
        });
        return record;
    };
```
객체를 넘겨주시면 이런 레포지토리 코드가 될 것 같습니다!